code/modules/events/masshallucination.dm

```diff
@@ -3,4 +3,17 @@
 
 /datum/event/mass_hallucination/start()
 	for(var/mob/living/carbon/C in living_mob_list)
-		C.hallucination += rand(100, 250)
\ No newline at end of file
+		C.hallucination += rand(100, 250)
+
+/datum/event/mass_drunk
+	startWhen = 10
+	announceWhen = 0
+
+/datum/event/mass_drunk/start()
+	for(var/mob/living/carbon/C in living_mob_list)
+		C.dizziness = 18
```

Comment:
This is the way the Subjugate spell works, I figured that was fine. I couldn't find any procs for it and I don't know how to specifically search for procs so I just went with this.